extensions/import/src/wizard/pages/modifyColumnsPage.ts

```diff
@@ -131,28 +130,35 @@ export class ModifyColumnsPage extends ImportPage {
 		await this.populateTable();
 		this.instance.changeNextButtonLabel(constants.importDataText);
 		this.loading.loading = false;
-
+		this.instance.registerNavigationValidator((info) => {
+			return this.table.data && this.table.data.length > 0;
+		});
 		return true;
 	}
 
 	override async onPageLeave(): Promise<boolean> {
+		await this.emptyTable();
 		this.instance.changeNextButtonLabel(constants.nextText);
+		this.instance.registerNavigationValidator((info) => {
```

Comment:
We should look into better ways of doing these navigation validators a lot - I see this pattern where we "clear" it by setting a new one that always returns true which doesn't seem the most clear. At least adding a "clearNavigationValidator" or something would seem to make sense.